shell/browser/api/electron_api_app.cc

```diff
@@ -1796,6 +1801,8 @@ gin::ObjectTemplateBuilder App::GetObjectTemplateBuilder(v8::Isolate* isolate) {
       .SetMethod("setAppLogsPath", &App::SetAppLogsPath)
       .SetMethod("setDesktopName", &App::SetDesktopName)
       .SetMethod("getLocale", &App::GetLocale)
+      .SetMethod("getPreferredSystemLanguages",
+                 &App::GetPreferredSystemLanguages)
```

Comment:
The `App::GetPreferredSystemLanguages` wrapper is not needed.

```suggestion
                 &GetPreferredLanguages)
```